### PR TITLE
Entry point for copay latest version number

### PR DESCRIPTION
**Pull request by bsnowden3**
Entry point to retrieve the latest version number of copay,  from github /releases/latest.

### CODE REVIEW

File: packages/bitcore-wallet-service/src/lib/expressapp.ts [hunk 1]
```diff
@@ -254,6 +255,21 @@ export class ExpressApp {
       };
     }
 
+    router.get('/latest-version', (req, res) => {
+      let server: WalletService;
+      try {
```
Comment (micahriggan) [inline packages/bitcore-wallet-service/src/lib/expressapp.ts]:
```
try {
  let server = getServer(req, res)
...
  const htmlString = await rp(latestCopayURL);
  const latestCopayVersion = htmlString.match(/Release v\d.\d.\d/);
  res.json({version: latestCopayVersion})
}
```

File: packages/bitcore-wallet-service/src/lib/expressapp.ts [hunk 2]
```diff
@@ -857,7 +872,7 @@ export class ExpressApp {
         });
       });
     });
-
+    //
```
Comment (micahriggan) [inline packages/bitcore-wallet-service/src/lib/expressapp.ts]: Not sure this comment line was needed

**Comment by cmgustavo** [comment]
IMO we should integrate it with the GitHub API instead of parse crude HTML : https://developer.github.com/v3/repos/releases/#get-the-latest-release

File: packages/bitcore-wallet-service/latest [hunk 3]
```diff
@@ -0,0 +1 @@
+<html><body>You are being <a href="https://github.com/bitpay/copay/releases/tag/v6.3.6">redirected</a>.</body></html>
```
Comment (matiu) [inline packages/bitcore-wallet-service/latest]: why we need this?

**Comment by matiu** [comment]
ACK, left a small comment regarding packages/bitcore-wallet-service/latest, why we need that file?

File: packages/bitcore-wallet-service/latest (hunk 3 above)
Comment (bsnowden3) [reply packages/bitcore-wallet-service/latest]: mistakenly added will clean

**Comment by matiu** [comment]
Could you add cache to this?  I am afraid this will be called to ofter, and could take a while.

Take a look at:

`checkAndUseGlobalCache (key, duration, cb)`

at `getFeeLevels` for example. it is very simple to use. A cache of ~2hrs would be fine. Thanks!

**Comment by matiu** [comment]
Code ACK. Please add tests.

File: packages/bitcore-wallet-service/test/expressapp.js [hunk 4]
```diff
@@ -50,7 +50,7 @@ describe('ExpressApp', function() {
       });
     });
 
-    describe('Routes', function() {
+    describe.only('Routes', function() {
```
Comment (nitsujlangston) [inline packages/bitcore-wallet-service/test/expressapp.js]: only

**Pull request merged**